### PR TITLE
Constant File Addition

### PR DESCRIPTION
**Pull request by AlariiHP**
We added a complete file including basic constants for the arm to achieve different heights in the arena. I hope mentors can consider this.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/BasketPos.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/BasketPos.java:1]: Good Work!

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java:1]: Check positions and fix conflicts

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Arm.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Arm.java:1]: Resolve conflicts

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Constants.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Constants.java:1]: Excelent!

File: build.dependencies.gradle
Comment (SantiQ0905) [inline build.dependencies.gradle:1]: Modify this.

**Review by SantiQ0905 — CHANGES_REQUESTED** [review]

**Review by AJahueyM — APPROVED** [review]

File: build.dependencies.gradle
Comment (SantiQ0905) [reply build.dependencies.gradle:1]: Alright

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java:1]: Fix merge

**Review by SantiQ0905 — CHANGES_REQUESTED** [review]

**Review by SantiQ0905 — APPROVED** [review]

**Pull request merged**